LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 moralmunky
```

Comment:
If these changes are acceptable I think you have to put your name here on the Copyright.

Reply:
Looked into the difference between open source licenses. I think MIT is fine. Do you want to do something like [Chromium](https://github.com/chromium/chromium/blob/master/LICENSE)?

`// Copyright 2015 The Chromium Authors. All rights reserved.`

Then list our names and email addresses in an AUTHORS file?

Reply:
That sounds fine to me.